sqllineage/core.py

```diff
@@ -201,13 +201,21 @@ def _handle_target_table_token(self, sub_token: TokenList) -> None:
                 self._lineage_result.write.add(Table.create(sub_token))
 
     def _handle_temp_table_token(self, sub_token: TokenList) -> None:
-        if not isinstance(sub_token, Identifier):
+        if isinstance(sub_token, Identifier):
+            self._lineage_result.intermediate.add(
+                Table.create(sub_token))
```

Comment:
I'm trying to enforce black code style. So according to black check, make this one line. 

You can see the black check result in GitHub actions (https://github.com/reata/sqllineage/pull/114/checks?check_run_id=1744343710) for this PR.
Or better, you can try adding [pre-commit](https://pre-commit.com/) hoot for your local git repo so each time before git commit, some sort of checks will be enforced before pushing to GitHub. I have a [pre-commit configuration file](https://github.com/reata/sqllineage/blob/master/.pre-commit-config.yaml) in the repo.